velox/functions/sparksql/Register.cpp

```diff
@@ -518,6 +518,8 @@ void registerFunctions(const std::string& prefix) {
       Varchar,
       Varchar,
       Varchar>({prefix + "mask"});
+  registerFunction<DateTruncFunction, Timestamp, Varchar, Timestamp>(
```

Comment:
Please move the function registration after the date time functions https://github.com/facebookincubator/velox/pull/11340/files#diff-19962809ab2250cbf3c99e92b7ba14ae89e2f26ae5e646db3c2b31e67af16e8dL450

Reply:
Moved, thanks.